internal/website/content/tutorials/guestbook.md

```diff
@@ -0,0 +1,308 @@
+---
+title: "Tutorial: Guestbook Application"
+linkTitle: "Guestbook"
+date: 2019-03-19T19:17:56-07:00
```

Comment:
Does this date need updating?

Reply:
I guess I assumed that this would get automatically updated somehow. I updated it manually for now.

@zombiezen ?